src/runners/helpers/db.py

```diff
@@ -145,7 +145,7 @@ def fetch(ctx, query=None, fix_errors=True, params=None):
             break
 
         def parse_field(value, field_type):
-            if value is not None and field_type['name'] in {
+            if value is not None and field_type.name in {
```

Comment:
Prior snowflake.connector.python==2.6.1 field_type was dictionary.
The field_type becomes tuple after 2.6.1.  The fix was to change the way to access name property.